src/shared/ui/Spinner.css

```diff
@@ -0,0 +1,75 @@
+.spinnerBlock {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+.spinner {
+    width: 80px;
+    height: 80px;
+}
+.spinnerBlock p {
+    margin-top: 40px;
+}
+
+.spinner .sk-cube {
```

Comment:
keep the same code styles. If there is an empty line between css blocks, then they should be everywhere